vocab/security/vocabulary.yml

```diff
@@ -527,55 +619,55 @@ property:
         url: https://tools.ietf.org/id/draft-multiformats-multibase-00.html
 
   - id: allowedAction
+    deprecated: true
     label: Allowed action
     comment: An action that the controller of a capability may take when invoking the capability.
     see_also:
       - label: Authorization Capabilities
         url: https://w3c-ccg.github.io/zcap-spec/#delegated-capability
 
-  - id: capability
-    label: Capability
-    comment: A cryptographic mechanism used to execute authority by possession.
-    see_also:
-      - label: Authorization Capabilities
-        url: https://w3c-ccg.github.io/zcap-spec/#capabilities-are-safer
-
   - id: capabilityChain
+    deprecated: true
     label: Capability chain
     comment: An list of delegated capabilities from a delegator to a delegatee.
     see_also:
       - label: Authorization Capabilities
         url: https://w3c-ccg.github.io/zcap-spec/#delegation
 
   - id: capabilityAction
+    deprecated: true
     label: Capability action
     comment: An action that can be taken given a capability.
     see_also:
       - label: Authorization Capabilities
         url: https://w3c-ccg.github.io/zcap-spec/#invoking-root-capability
 
   - id: caveat
+    deprecated: true
     label: Caveat
     comment: A restriction on the way the capability may be used.
     see_also:
       - label: Authorization Capabilities
         url: https://w3c-ccg.github.io/zcap-spec/#caveats
 
   - id: delegator
+    deprecated: true
     label: Delegator
     comment: An entity that delegates a capability to a delegatee.
     see_also:
       - label: Authorization Capabilities
         url: https://w3c-ccg.github.io/zcap-spec/#delegation
 
   - id: invocationTarget
+    deprecated: true
     label: Invocation target
     comment: An invocation target identifies where a capability may be invoked, and identifies the target object for which the root capability expresses authority.
     see_also:
       - label: Authorization Capabilities
         url: https://w3c-ccg.github.io/zcap-spec/#root-capability
 
   - id: invoker
+    deprecated: true
```

Comment:
Most of this isn't deprecated, as it's a part of the Authorization Capabilities work: https://w3c-ccg.github.io/zcap-spec/

Reply:
If it is there then the vocabulary must be there in a separate document. This vocabulary is for terms defined in this specification and this specification only imho.

Reply:
Not sure how we'll want to resolve this, but we do need to keep the terms around somehow and have a way to incubate new terms.